src/test/java/spoon/test/issue4357/TestIssue4357.java

```diff
@@ -0,0 +1,33 @@
+package spoon.test.issue4357;
+
+import org.junit.jupiter.api.Test;
+import spoon.Launcher;
+import spoon.reflect.CtModel;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestIssue4357 {
+    
+    @Test
+    public void testClasspathURLWithSpaces() throws MalformedURLException {
```

Comment:
Required change: Put this test in an existing test class instead of creating a new one. Due to the phrasing of the contract, I'd put this in `LauncherTest`.

Reply:
@slarse done.